notebooks/biodiversity_mapping.ipynb

```diff
@@ -7,7 +7,7 @@
         "id": "view-in-github"
       },
       "source": [
-        "<a href=\"https://colab.research.google.com/github/ayushanand18/pyobis/blob/bio-mapping/notebooks/biodiversity_mapping.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+        "<a href=\"https://colab.research.google.com/github/iobis/pyobis/blob/master/notebooks/biodiversity_mapping.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
I think that both @7yl4r and I have the power to rename the master branch to main now. Maybe let's take the opportunity do to this now and change these links only once?

Reply:
good call! I just renamed the default branch from `master` to `main`. Please reconfigure your remote repos accordingly!

Reply:
Update. I have updated the badges `master`->`main`.